<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="24638" started-at="2018-07-23T02:16:28Z" finished-at="2018-07-23T02:16:53Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="24638" started-at="2018-07-23T02:16:28Z" finished-at="2018-07-23T02:16:53Z">
      <class name="org.test.automation.test.GmailTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:org.test.automation.test.GmailTest@337d0578]" name="setup" is-config="true" duration-ms="4256" started-at="2018-07-22T22:16:28Z" finished-at="2018-07-22T22:16:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="testGmailLogin()[pri:0, instance:org.test.automation.test.GmailTest@337d0578]" name="testGmailLogin" duration-ms="17606" started-at="2018-07-22T22:16:33Z" finished-at="2018-07-22T22:16:50Z">
          <exception class="org.test.automation.exception.GmailException">
            <message>
              <![CDATA[Locator not available: By.id: identifierId1]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.test.automation.exception.GmailException: Locator not available: By.id: identifierId1
	at org.test.automation.base.Helper.getelement(Helper.java:25)
	at org.test.automation.base.Helper.enterText(Helper.java:39)
	at org.test.automation.test.GmailLogin.enterGmailUserName(GmailLogin.java:19)
	at org.test.automation.test.GmailTest.testGmailLogin(GmailTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.test.automation.exception.GmailException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGmailLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:org.test.automation.test.GmailTest@337d0578]" name="tearDown" is-config="true" duration-ms="2731" started-at="2018-07-22T22:16:50Z" finished-at="2018-07-22T22:16:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- org.test.automation.test.GmailTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
